DEVELOPMENT.md

```diff
@@ -15,18 +15,36 @@ The interface provides two functions for generating resources: `generate` and `g
 A feature will normally only need to implement the `generate` function which will be executed in parallel.
 The `generateSequentially` function is intended for resources where parallel generation may result in duplicate resources (for example shared databases [DatabaseFeature](./src/main/kotlin/no/skatteetaten/aurora/boober/feature/DatabaseFeature.kt)). 
 
+The functions that can be implemented, in running order, are:
+
+| #   | function                          | return value                                                                         | desc                                                                                          |
+|-----|-----------------------------------|--------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
+| 1   | `enable`                          | `bool`                                                                               | indicates if feature should run                                                               |
+| 2   | `handlers`                        | `Set<AuroraConfigFieldHandler>`                                                      | create handlers for configuration fields                                                      |
+| 3   | `createContext`                   | [FeatureContext](./src/main/kotlin/no/skatteetaten/aurora/boober/feature/Feature.kt) | create context for a given feature, context is sent to `validate`, `generate`, `modify` steps |
```

Comment:
vi kan nevne at createContext burde kun brukes for operasjoner som tar lang tid. Hvis ikke så er det ikke vits.

Reply:
la til en parentes om at dette kun burde brukes på tidkrevende operasjoner